source/templates/layouts/default.hbs

```diff
@@ -37,6 +37,16 @@
   {{body}}
 
   <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
@cromwellryan do we want this here or right before the closing `head` tag? https://support.google.com/analytics/answer/1008080